demos/list.scss

```diff
@@ -75,6 +76,22 @@ a.material-icons {
   margin-left: 72px;
 }
 
+.checkbox-list-ripple-surface {
+  @include mdc-ripple-surface;
+  @include mdc-ripple-radius;
+  @include mdc-states;
+  @include mdc-states-base-color(black);
+
+  position: relative;
```

Comment:
I think this property and `overflow: hidden` are already being set by `mdc-list-item`.

Is there a specific reason you needed to add them?

Reply:
I didn't try taking them off. I copied from mdc-ripple-surface. Let me try removing